Docker testing: on Linux build fails with "not a directory" error

When I try to run current integration tests (with docker) I get following exception: 

```
[ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.15.16:build (start) on project DataCleaner-monitor-ui: Unable to build image [datacleaner-monitor]: stat /var/lib/docker/aufs/mnt/481a3b0da2b      ffa9376b48855ddf22246a50a6b6b7bc37a646b18d540a7023f23/usr/local/tomcat/webapps/DataCleaner-monitor-ui-5.1.4-SNAPSHOT.war: not a directory -> [Help 1]
```

--- a/monitor/ui/src/test/resources/Dockerfile
+++ b/monitor/ui/src/test/resources/Dockerfile
@@ -3,4 +3,4 @@ FROM tomcat:8-jre8
 # The docker-maven-plugin adds the war to the maven folder because of the result of the <assembly> element within
 # <build> is added to the created docker image. Files and artifacts contained in the assembly are by default copied
 # under the "/maven" directory.
-COPY maven/${project.build.finalName}.war /usr/local/tomcat/webapps
+COPY maven/${project.build.finalName}.war /usr/local/tomcat/webapps/${project.build.finalName}.war

